components/21-atoms/table/_table.scss

```diff
@@ -0,0 +1,105 @@
+div.responsive-table,
+div.table-list {
+  .list-description,
+  caption {
+    @include theme('color', 'color-tertiary--lighten-1', 'table-caption-color');
+
+    margin: .4rem 0;
+    font-size: .6rem;
+    text-align: left;
+  }
+}
+
+div.responsive-table {
+  @include tablet {
+    display: block;
+  }
+
+  display: none;
+  overflow-x: auto;
+
+  &:focus {
+    border: 2px dotted;
```

Comment:
has this been designed?
We really need a focus style (looking at you buttons!), so perhaps it's the right moment to move this to a mixin or apply it base to a bunch of focusable elements at once.

Reply:
No it has not... but I'd do this as a separate issue because it impacts more then just the table component... But you are right, we should add a ticket for a focus style, it should be designed and we should prioritize it...

Reply:
Added a ticket for this: https://digipolisgent.atlassian.net/browse/DTGB-715

Reply:
Just one problem... there is no styling yet... but okay I'll create a mixin with border: 2px dotted in then...